From b9b4d16ee422f0a731698b90316f0b2dc118027e Mon Sep 17 00:00:00 2001
From: guixxx <guihkx@gmail.com>
Date: Sat, 14 Apr 2018 10:15:19 -0300
Subject: [PATCH] fix url for binfmt documentation (#8720)

this file went unnoticed in PR #6223

(cherry picked from commit de03b2951263892c8670c3f8ee81ea9b8b48e504)
---
 units/systemd-binfmt.service.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/units/systemd-binfmt.service.in b/units/systemd-binfmt.service.in
index df9396d895..e940c7c9ad 100644
--- a/units/systemd-binfmt.service.in
+++ b/units/systemd-binfmt.service.in
@@ -10,7 +10,8 @@
 [Unit]
 Description=Set Up Additional Binary Formats
 Documentation=man:systemd-binfmt.service(8) man:binfmt.d(5)
-Documentation=https://www.kernel.org/doc/Documentation/binfmt_misc.txt
+Documentation=https://www.kernel.org/doc/html/latest/admin-guide/binfmt-misc.html
+Documentation=https://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
 DefaultDependencies=no
 Conflicts=shutdown.target
 After=proc-sys-fs-binfmt_misc.automount
-- 
2.17.0

